src/targets/github.ts

```diff
@@ -228,11 +209,12 @@ export class GitHubTarget extends BaseTarget {
     const assets = await this.getAssetsForRelease(release_id);
     const assetToDelete = assets.find(({ name }) => name === assetName);
     if (!assetToDelete) {
-      throw new Error(
-        `No such asset with the name "${assetToDelete}". We have these instead: ${assets.map(
+      logger.warn(
+        `No such asset with the name "${assetToDelete}", moving on. We have these instead: ${assets.map(
           ({ name }) => name
         )}`
       );
+      return false;
```

Comment:
I'd move listing of assets to a `logger.trace` as it can be very noisy as a warning.

Reply:
I left it as `warn` to see if we stumble upon this branch now. We can move it to `trace` once this issue is resolved.